Source/TcpSocketPlugin/Public/TcpSocketConnection.h

```diff
@@ -72,34 +72,116 @@ class TCPSOCKETPLUGIN_API ATcpSocketConnection : public AActor
 	UFUNCTION(BlueprintPure, meta = (DisplayName = "Append Bytes", CommutativeAssociativeBinaryOperator = "true"), Category = "Socket")
 	static TArray<uint8> Concat_BytesBytes(TArray<uint8> A, TArray<uint8> B);
 
+	/** Converts a byte to an array of bytes */
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Byte To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
+	static TArray<uint8> Conv_ByteToBytes(uint8 InByte);
+
+	/** Converts a short to an array of bytes */
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Short To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
+	static TArray<uint8> Conv_ShortToBytes(int16 InShort)
+	{
+		TArray<uint8> byteArray;
+		byteArray.Add((InShort >> 8) & 0xFF);
+		byteArray.Add(InShort & 0xFF);
+		return byteArray;
+	}
+
 	/** Converts an integer to an array of bytes */
 	UFUNCTION(BlueprintPure, meta = (DisplayName = "Int To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
 	static TArray<uint8> Conv_IntToBytes(int32 InInt);
 
-	/** Converts a string to an array of bytes */
-	UFUNCTION(BlueprintPure, meta = (DisplayName = "String To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
-	static TArray<uint8> Conv_StringToBytes(const FString& InStr);
+	/** Converts a long to an array of bytes */
+	UFUNCTION(BlueprintPure, meta = (DisplayName = "Long To Bytes", CompactNodeTitle = "->", Keywords = "cast convert", BlueprintAutocast), Category = "Socket")
+	static TArray<uint8> Conv_LongToBytes(int64 InLong)
```

Comment:
Please move the function definition into .cpp
And unroll the loop into an in-place initialization